guest.speaking/MedicalDoctors4CovidEthicsIntl-CharlesKovess/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess]/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt

```diff
@@ -961,13 +961,13 @@ WEBVTT
 [J Jay Couey]: And I even based my understanding of the brain and my organization of my thoughts on how to pursue a further understanding of the brain based on this idea that I had to think of neurons as expressing
 
 40:26.062 --> 40:33.732
-[J Jay Couey]: Jeans and jeans coming on and off and how even though we can't monitor that we assume it's happening and all of this gets fueled by these.
+[J Jay Couey]: genes and genes coming on and off and how even though we can't monitor that we assume it's happening and all of this gets fueled by these.
```

Comment:
W/ the way modern biology is going, there might not be any difference between jeans and genes in significance... :sweat_smile:

Reply:
True.